### ISSUE TITLE
请教问题

### ISSUE THREAD

**Issue by leileilin** [WZMIAOMIAO/deep-learning-for-image-processing#439]
你好，我看你在多gpu的代码只是对每一轮的模型进行保存，并没有保存最优的模型，我在你的代码上增加了对val_loss的大小判断选择最优模型，但是即使上一轮val_loss = 0.9 这一轮val_loss=0.6，但是我的best_score不知道为什么会等于0.6，导致无法选择最优模型，你知道是什么原因吗？
`if rank == 0:
            score = eval_loss
            if score < best_score:
                print("Saving Best ..")
                print("Validation Loss: %.6f" % score)
                print("Best Score: %.6f" % best_score)
                save_model(model.module if hasattr(model, "module") else model, tokenizer, os.path.join(model_config.save_model_path, "checkpoint_best/"))
                best_score = score
                print("Best Score: %.6f" % best_score)
                patience = 0 
            else:
                print("Validation Loss: %.6f" % score)
                print("Best Score: %.6f" % best_score)
                patience += 1`

非常感谢！！！
